software-requirement-specification.md

```diff
@@ -71,25 +79,37 @@ This subsection should:
 3. Specify the sources from which the references can be obtained.
 This information may be provided by reference to an appendix or to another document.
 
-## 2. General Description
+Which format is for the references (?)
```

Comment:
is this intended to be left in?

Reply:
lgtm otherwise

Reply:
No, thats a question I have for the TA and i dont wanna forget lol